test/openapi3/general/external-ref/file-refs.test.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const chai = require('chai').use(require('chai-as-promised'));
+const path = require('path');
+const fs = require('fs');
+const jsyaml = require('js-yaml');
+const { expect } = chai;
+
+const schemaValidatorGenerator = require('../../../../src');
+
+describe('Loading defintions file with file refs', () => {
+    describe('calling buildSchemaSync with path to defintions file', () => {
```

Comment:
defintions -> definitions (couple of times in this file)

Reply:
fixed